examples/rehastimp24_example.py

```diff
@@ -0,0 +1,170 @@
+from pyScienceMode2.channel import Channel, Point
+from pyScienceMode2.rehastim_interface import StimulatorP24 as St
+
+"""
+This example shows how to use the RehastimP24 device. 
+There are several commands divided into three levels (general, low and mid).
+You can't call commands from different levels, you must first close the current level
+to be able to use commands from another one.
+"""
+
+# list which contains the channels you want to use
+list_channels = []
+
+# list which contains the points you want to use
+list_stimulation_points = []
+
+# Create an object channel
+channel_1 = Channel(
+    no_channel=1, name="Biceps", amplitude=40.1, pulse_width=500, frequency=35, device_type="RehastimP24"
+)
+channel_2 = Channel(
+    mode="Doublet",
+    no_channel=2,
+    amplitude=40,
+    pulse_width=500,
+    name="Triceps",
+    frequency=35,
+    ramp=5,
+    device_type="RehastimP24",
+)
+channel_3 = Channel(
+    mode="Triplet",
+    no_channel=3,
+    amplitude=40,
+    pulse_width=500,
+    name="Triceps",
+    frequency=15,
+    ramp=15,
+    device_type="RehastimP24",
+)
+
+# Create an object stimulator
+stimulator = St(port="COM4", show_log="Partial")  # Enter the port on which the rehastim is connected
+
+# Add the channels you want to stimulate to the list.
+list_channels.append(channel_1)
+list_channels.append(channel_2)
+# list_channels.append(channel_3)
+
+"""
+General level commands. 
+In this level you can get several information about the device.
+"""
+
+# stimulator.get_battery_status()
+# stimulator.get_stim_status()
+# stimulator.get_main_status()
+# stimulator.get_all()
+# stimulator.reset()
+
+"""
+Low level commands.
```

Comment:
In this level you can do the actual stimulation

Reply:
Please be more specific

Reply:
Forget about it